test/test_service.py

```diff
@@ -12,6 +12,25 @@
 
 # Samples that we will be sending to the service
 sample1 = dict(
+    sid=1,
+    metadata={},
+    service_name='configextractor',
+    service_config={},
+    fileinfo=dict(
+        magic='ASCII text, with no line terminators',
+        md5='fda4e701258ba56f465e3636e60d36ec',
+        mime='text/plain',
+        sha1='af2c2618032c679333bebf745e75f9088748d737',
+        sha256='c805d89c6d26e6080994257d549fd8fec2a894dd15310053b0b8078064a5754b',
+        size=19,
+        type='unknown',
+    ),
+    filename='c805d89c6d26e6080994257d549fd8fec2a894dd15310053b0b8078064a5754b',
```

Comment:
either we call this filename "sample_txt" or we change filename='mirai_strings' -> filename=<sha-256> of file. your call

Reply:
changed mirai_strings to sha256